build.sbt

```diff
@@ -31,7 +31,7 @@ addCommandAlias("benchQuick", benchQuick.command)
 // it runs `test` sequentially in every defined module.
 commands += Command.command("ci-fast") { s =>
   if (ciScalaVersion.startsWith("2.10")) {
-    s"wow $ciScalaVersion" :: "tests210/test" :: s
+    s"wow $ciScalaVersion" :: "langmetaTestsJVM/test" :: s
```

Comment:
`plz $ciScalaVersion test` should do this for you ;) but I don't mind as long as it works!